.github/workflows/stale.yml

```diff
@@ -0,0 +1,17 @@
+name: "Close stale PRs"
+on:
+  schedule:
+    - cron: "0 23 * * *"
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v3
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          # Empty message to prevent issues from being closed or staled
+          stale-issue-message: ''
+          stale-pr-message: 'The PR is marked as stale since no activity has been recorded'
```

Comment:
It would be informative for PR authors to be notified on the amount of expiration days in the message. Can this be added?

Reply:
done!